mythril/laser/ethereum/state.py

```diff
@@ -183,14 +214,14 @@ class MachineState:
     """
     MachineState represents current machine state also referenced to as \mu
     """
-    def __init__(self, gas):
+    def __init__(self, gas, pc=0, stack=[], memory=[], constraints=None, depth=0):
```

Comment:
There are mutable defaults here. You could use: memory = None
and later
self.memory = memory or []

Reply:
oops, this I forgot, this PR was older than that one :sweat_smile:

Reply:
bump

Reply:
@norhh This is still mutable